src/main/java/com/zhongan/devpilot/actions/editor/popupmenu/PopupMenuEditorActionGroupUtil.java

```diff
@@ -115,28 +116,30 @@ protected void actionPerformed(Project project, Editor editor, String selectedTe
 
                         EditorInfo editorInfo = new EditorInfo(editor);
                         if (editorActionEnum == EditorActionEnum.GENERATE_TESTS) {
-                                if (language != null && language.isJvmPlatform()
-                                        && PsiFileUtil.isCaretInWebClass(project, editor)) {
-                                    data.put(ADDITIONAL_MOCK_PROMPT, PromptConst.MOCK_WEB_MVC);
+                            if (language != null && language.isJvmPlatform()
+                                    && PsiFileUtil.isCaretInWebClass(project, editor)) {
+                                data.put(ADDITIONAL_MOCK_PROMPT, PromptConst.MOCK_WEB_MVC);
+                            }
+                            if (language != null && "java".equalsIgnoreCase(language.getLanguageName())) {
+                                UtFrameworkProvider utFrameworkProvider = UtFrameworkProviderFactory.create("java");
```

Comment:
Since the provider will judge the language, it's better to create before the language judge